dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_mysql.sql

```diff
@@ -2007,3 +2010,23 @@ CREATE TABLE `t_ds_fav_task`
 ) ENGINE = InnoDB
   AUTO_INCREMENT = 1
   DEFAULT CHARSET = utf8;
+
+-- ----------------------------
+-- Table structure for t_ds_task_remote_host
+-- ----------------------------
+DROP TABLE IF EXISTS `t_ds_task_remote_host`;
+CREATE TABLE `t_ds_task_remote_host`
+(
+    `id` int(11) NOT NULL AUTO_INCREMENT COMMENT 'id',
+    `code` bigint(20) NOT NULL DEFAULT '0' COMMENT 'encoding',
```

Comment:
It's better to add indexes in this table. So as postgresql.

Reply:
> It's better to add indexes in this table. So as postgresql.

Sure.